Real-Time Weather Monitoring System/Weather/Readers/Factory/WeatherReaderFactory.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Real_Time_Weather_Monitoring_System.Weather.models;
+using System.Text;
+
+namespace Real_Time_Weather_Monitoring_System.Weather.Readers.Factory;
+
+public class WeatherReaderFactory : IWeatherReaderFactory
+{
+
+    public IWeatherData? GetWeatherData(StringBuilder sb)
+    {
+        var data = sb.ToString();
+        IWeatherReader reader;
+        if (data.Trim().Contains("{"))
```

Comment:
Nit can we have private method or at least variable so the condition becomes more readable